src/App.js

```diff
@@ -1,20 +1,30 @@
 import './App.css';
-import { Routes, Route } from 'react-router-dom'
+import { Routes, Route } from 'react-router'
 import Header from './components/Header'
 import NavBar from './components/NavBar'
 import ReviewList from './components/ReviewList'
-import { useState } from 'react'
+import { useEffect, useState } from 'react'
+import { getAllCategories } from './utils/API'
 
 function App() {
   const [reviewByID, setReviewByID] = useState();
+  const [allCategories, setAllCategories] = useState([]);
+
+useEffect(() => {
+  getAllCategories().then((categoriesFromAPI) => {
+    setAllCategories(categoriesFromAPI)
+  })
+}, [])
 
   return (
     <div className="App">
-    <Header />  
-    <NavBar />  
-    <Routes>
-      <Route path="/"  element={<ReviewList reviewByID={reviewByID} setReviewByID={setReviewByID}/>} />
-    </Routes>
+      <Header />
+      <div className='NavHome'>
+        <NavBar allCategories={allCategories}/>
+      </div>
+      <Routes>
+        <Route path="/reviews" element={<ReviewList reviewByID={reviewByID} setReviewByID={setReviewByID} />} />
```

Comment:
Also could the review by id be lowered? There's a way you could integrate it into the review list component. If you're not sure how feel free to pop in an NC Help tomorrow morning :)